src/toplevel.c

```diff
@@ -135,7 +135,9 @@ static jl_value_t *jl_eval_module_expr(jl_module_t *parent_module, jl_expr_t *ex
     }
 
     int is_parent__toplevel__ = jl_is__toplevel__mod(parent_module);
-    jl_module_t *newm = jl_new_module(name, is_parent__toplevel__ ? NULL : parent_module);
+    // If we have `Base`, don't also try to import `Core` - the `Base` exports are a superset and we
+    // avoid ambiguities as to which module the bindings should come from.
```

Comment:
```suggestion
    // If we have `Base`, don't also try to import `Core` - the `Base` exports are a superset
```

This statement does not seem consistent with the definition of exports, since something cannot be ambiguous if it has the same value.

Reply:
The binding isn't ambiguous, but it's ambiguous where the binding came from, which we print in various places and people complain.

Reply:
Ah, might help to update the wording then to be clear that the ambiguity is just the printing of the module name, not the value?